config/locales/metadata/en.yml

```diff
@@ -436,7 +436,7 @@ en:
 
         data_release_delay_reason:
           label: Reason for delaying release
-          help: "To apply for a delay, please send the following information to <%= configatron.data_sharing_contact.name %> (<%= configatron.data_sharing_contact.email %>):<ul><li>SAC sponsor</li><li>Study title and, where available, data set(s) ID</li><li>Study description (should describe the data types that will be produced)</li><li>Data sharing plan (to include the following):<ul><li>- Which repository/repositories will be used</li><li>- If data will be made available under a managed access mechanism</li><li>- When data will be shared</li></ul></li><li>Reason(s) for delaying data release</li></ul>"
+          help: 'To apply for a delay which is not associated with a PhD study, please contact <a href="mailto:<%= configatron.data_sharing_contact.email %>"><%= configatron.data_sharing_contact.email %></a>'
```

Comment:
This is where i'm seeing you dropped the reference to `configatron.data_sharing_contact.name` entirely and replaced it with two instances of `configatron.data_sharing_contact.email` so I don't think the name is used unless it's somewhere else.